sw-manifest.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+require('dotenv').config();
+const isProd = process.env.ELEVENTY_ENV === 'prod';
+
+const log = require('fancy-log');
+const {promises: fs} = require('fs');
+const {getManifest} = require('workbox-build');
+const resourcePath = require('./src/build/resource-path');
+
+process.on('unhandledRejection', (reason, p) => {
+  log.error('Build had unhandled rejection', reason, p);
+  throw new Error(`Build had unhandled rejection ${reason}`);
+});
+
+/**
+ * Builds the cache manifest for inclusion into the Service Worker.
+ *
+ * TODO(samthor): This relies on both the gulp and CSS tasks occuring
```

Comment:
Might remove the TODO and just leave this as a comment. I think since workbox has to scan the dist/ directory it's sort of unavoidable that our other build tasks will have to run first.